src/test/java/org/threadly/concurrent/AbstractPrioritySchedulerTest.java

```diff
@@ -0,0 +1,375 @@
+package org.threadly.concurrent;
```

Comment:
Shared unit tests from PrioritySchedulerTest were abstracted into this.  This gets us SingleThreadScheduler priority tests for almost no effort.